src/index.ts

```diff
@@ -131,9 +131,9 @@ function getTypeGuardName(
     for (const line of doc.getInnerText().split('\n')) {
       const match = line
         .trim()
-        .match(/@see\s+(?:{\s*@\w+\s*(\w+)\s*}\s+)?ts-auto-guard:([^\s]*)/)
+        .match(/@see\s+(?:{\s*(@link\s*)?(\w+)\s*}\s+)?ts-auto-guard:([^\s]*)/)
```

Comment:
FYI you can use a non-capturing group here `(?:@link\s*)` and it should be `\s+` since you can't do `@linkfoo`.